org.godotengine.GodotSharp.yaml

```diff
@@ -0,0 +1,141 @@
+app-id: org.godotengine.GodotSharp
```

Comment:
Shouldn't this be `org.godotengine.Godot4Sharp`?

Reply:
So how does it work with distro installed version? Seems like a bug if it dumps dotNet into the product even if not used.